test-util/src/main/java/io/camunda/zeebe/test/util/WorkloadGenerator.java

```diff
@@ -93,6 +91,12 @@ public static void performSampleWorkload(final ZeebeClient client) {
         .join();
 
     // wait for incident and resolve it
+    TestUtil.waitUntil(
```

Comment:
:x: This is old code, let's use `Awaitility` for this.

Reply:
Awaitility needs to be added as a dependency. Do you know what the correct scope would be? It's not `test` but I'm not sure if it should be `compile`, `runtime` or `provided`.

Reply:
Since test-util is an internal module for testing utilities for Zeebe, it's perfectly fine as compile. I guess technically it could be provided, but we're not very strict for internal testing modules since they're not meant to be consumed by any users and don't end up in our distribution.